src/components/emailLogin.vue

```diff
@@ -67,132 +67,95 @@
 }
 </i18n>
 
-<script>
-	export default {
+<script lang="ts">
+declare var grecaptcha: any;
+import { defineComponent } from 'vue'
+import { mapGetters, mapActions, mapMutations } from 'vuex';
+	export default defineComponent({
 		data() {
 			return {
-				errors: null,
```

Comment:
Don't know why it look like this. Anyway, I think we will use PRETTIER later, so it doesn't matter